ui/app/AppLayouts/Wallet/panels/WalletHeader.qml

```diff
@@ -73,6 +75,53 @@ Item {
             spacing: 16
             Layout.alignment: Qt.AlignTrailing
             Layout.topMargin: 5
+            Row {
+                Layout.alignment: Qt.AlignRight | Qt.AlignTop
+                Layout.preferredHeight: 38
+                spacing: 8
+
+                StatusButton {
+                    id: reloadButton
+                    size: StatusBaseButton.Size.Tiny
+                    loadingIndicatorSize: size
+                    height: parent.height
+                    width: height
+                    borderColor: Theme.palette.directColor7
+                    borderWidth: 1
+
+                    normalColor: Theme.palette.transparent
+                    hoverColor: Theme.palette.baseColor2
+
+                    icon.name: "refresh"
+                    icon.color: hovered ? Theme.palette.directColor1 : Theme.palette.baseColor1
+                    asset.mirror: true
+
+                    loading: RootStore.walletSectionInst.isAccountTokensReloading
+                    interactive: !loading && !throttleTimer.running
+                    readonly property string lastReloadTime: !!RootStore.walletSectionInst.lastReloadTimestamp ?
+                                              LocaleUtils.formatRelativeTimestamp(
+                                                            RootStore.walletSectionInst.lastReloadTimestamp * 1000) : ""
```

Comment:
`RootStore.walletSectionInst.lastReloadTimestamp` should be actually put into the store as sth like:
```qml
readonly property double lastReloadTimestamp: walletSectionInst.lastReloadTimestamp
```
because that way it can be mocked in StoryBook and/or tests, and you're not risking the overflow (`int` is simply not enough for the timestamp on QML side)